src/foam/nanos/cron/CronSchedulerService.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.INTERFACE({
+  package: 'foam.nanos.cron',
+  name: 'CronSchedulerService',
```

Comment:
What is this needed for?

Reply:
In my understanding, the `CronSchedulerService` was necessary because of its `run` method that contains a loop which verifies any cronjobs that must run at a given time. 
But I could be wrong, perhaps it doesn't need its own service? 

The original class `CronSchedulerService` author could clarify that.

Reply:
@diaswrd Yeah I don't see why this service is necessary to be honest. CronScheduler.java implements NanoService. The run method is there because it implements Runnable.

Reply:
@diaswrd Was this done so you could auto generate the Skeleton?

Reply:
In any case, I would remove the run method and add
`javaExtends: [ 'java.lang.Runnable' ]` instead